script/Arcueid/Day 4/04_00_ARC04_1B.txt

```diff
@@ -151,12 +162,13 @@ After leaving the hospital, the doctor told me to avoid any hard to digest foods
 [sha:544730ca86e6d875a103fbed05fcc9dda7f1541e]{
 -- Page 38, Offset 5659.
 -- それでしたら安心してお食事を用意できます。さ、すぐにお作りしますから、居間で待っていてくださいな」
-If that is the case, I can prepare your meal without any worries. Please wait for me in the living room while I get it ready."
+In that case, it sounds like I don't need to worry about anything while preparing your meal.
+ Alright, I'll whip something up for your right away, so please wait in the living room."
 }
 [sha:c54d516c174ea07176e6d137ee3429a809eae82c]{
 -- Page 39, Offset 5660.
 -- 琥珀さんは忙しそうに居間へ踵を返す。
-Kohaku-san turns her back and heads to the living room, looking busy.
+Kohaku-san wheels around and starts scurrying back toward the living room.
```

Comment:
not a fan of "wheels around". It might not have the same instantaneous quality but just "Kohaku-san turns around and" sits a whole lot better with me. Much more cliche but you could also go for "turns on a dime"

Reply:
Spins around? Does an about-face?

Reply:
spins around is fine by me